gapic/batch/common.yaml

```diff
@@ -0,0 +1,5 @@
+common:
+  batch_apis: '*'
+  batch_languages: '*'
+  api_config_pattern: ${THISDIR}/../api/artman_*.yaml
+  artman_language_yaml: ${THISDIR}/../lang/common.yaml
```

Comment:
Sorry, one last thing - use ${REPOROOT}/googleapis/gapic/api/* instead of ${THISDIR}/../api/* (likewise for the /lang/ case). We have removed the relative paths elsewhere.